autorest/adal/msi_windows.go

```diff
@@ -0,0 +1,11 @@
+// +build windows
+
+package adal
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// msiPath is the path to the MSI Extension settings file (to discover the endpoint)
+var msiPath = filepath.Join(os.Getenv("SystemDrive"), "WindowsAzure/Config/ManagedIdentity-Settings")
```

Comment:
When I run this on my machine I get the following, which is missing a slash!

`C:WindowsAzure\Config\ManagedIdentity-Settings`

Reply:
fixed